package.json

```diff
@@ -94,6 +94,7 @@
     "jsdom-global": "3.0.2",
     "nyc": "11.7.1",
     "react-dom": "16.3.2",
+    "react-native-browser-polyfill": "0.1.2",
```

Comment:
Why do we need this?

Reply:
Oops yea. it is not needed anymore. let me remove that. Thanks